packages/core/src/components/select/_select.scss

```diff
@@ -44,11 +46,15 @@
       border-radius: 2px;
 
       [dir='rtl'] & {
-        position: unset;
-        margin-left: $ray-field-h-spacing;
+        right: calc(100% - (2 * #{$ray-field-h-spacing}));
```

Comment:
This is a slight change in how we handle the arrow on RTL. Instead of unsetting position, we calculate it so that the behavior is similar to how it is done with LTR